packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2995,10 +3032,13 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
                 totalChunks: chunkN,
             };
             offset += maxOpSize;
+            const serializedChunkedOp = JSON.stringify(chunkedOp);
```

Comment:
> JSON.stringify [](http://example.com/codeflow?start=39&length=15)

I wish we could not pay this price in case we will not compress it anyways - Like validate the maxOpSize vs our min compression value and pass empty string otherwise.

Reply:
Just pushed a change that should only stringify if we're going to compress it.